test/passing/tests/js_source.ml.ocp

```diff
@@ -180,12 +180,14 @@ class%foo x = x [@@foo]
 class type%foo x = x [@@foo]
 external%foo x : _ = "" [@@foo]
 exception%foo X [@foo]
-module%foo M = M [@@foo]
 
-module%foo rec M : S = M [@@foo]
-and M : S = M [@@foo]
+module%foo [@foo] M = M
+
+module%foo [@foo] rec M : S = M
+and [@foo] M : S = M
+
+module type%foo [@foo] S = S
```

Comment:
This regression is not due to this PR and is being fixed here: https://github.com/ocaml-ppx/ocamlformat/pull/2450